dev-docs/bookshelf-sigv4-flow.txt

```diff
@@ -0,0 +1,50 @@
+Bookshelf Sigv4 Authentication

```

Comment:
Could you also link this doc int he dev-docs/README.md?

Reply:
I linked it in the `BOOKSHELF.md` doc you asked me to create above (I assume that's more appropriate?)